app/services/stats/r901_offender_sar_cases_report.rb

```diff
@@ -28,27 +28,27 @@ def self.description
 
     def initialize(**options)
       super(**options)
-      @case_scope = options[:case_scope] || Case::SAR::Offender.all
-    end 
+      @case_scope = options[:case_scope] || Case::Base.offender_sar.all
+    end
 
     def case_scope
-      @case_scope.where(type: 'Case::SAR::Offender')
+      @case_scope.where(type: 'Case::SAR::Offender').or(@case_scope.where(type: 'Case::SAR::OffenderComplaint'))
```

Comment:
It will cause potential problem if we use `or` in this way when the filter options are passed into this report 
I would suggest to use the following one 
`@case_scope.where(type: ["Case::SAR::Offender", "Case::SAR::OffenderComplaint"])`